python/friesian/src/bigdl/friesian/feature/table.py

```diff
@@ -1004,31 +1000,66 @@ def hash_encode(self, columns, bins, method='md5'):
             hash_df = hash_df.withColumn(col_name, hash_int(pyspark_col(col_name)))
         return FeatureTable(hash_df)
 
-    def cross_hash_encode(self, columns, bins, cross_col_name=None, method='md5'):
+    def cross_hash_encode(self, column_pairs, bin_sizes, crossed_col_names=None, method='md5'):
```

Comment:
is the `column_pair` name reasonable? We can do cross on more than 2 columns right?

Reply:
what about `cross_columns`?